lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -10,7 +10,6 @@ class Base
           ::Errno::ENOENT,
           ::Racc::ParseError,
           ::RubyParser::SyntaxError,
-          ::RuntimeError,
```

Comment:
This seems pretty unrelated to this change?

This was added as a [bugfix](https://github.com/codeclimate/codeclimate-duplication/commit/6715ad152d95c3533e2722b3fd6e716354fb3616) because some ruby parser errors are thrown as `RuntimeException` instances, and we want to catch those.

It's very unusual for thrown exceptions to be `RuntimeError` instances: usually such errors would inherit from `RuntimeError`, which is why the `rescue` clause in `#run` will re-raise errors that are not this exact class.